src/controller/artifact/processor/sbom/sbom.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sbom
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+
+	"github.com/goharbor/harbor/src/controller/artifact/processor"
+	"github.com/goharbor/harbor/src/controller/artifact/processor/base"
+	"github.com/goharbor/harbor/src/lib/errors"
+	"github.com/goharbor/harbor/src/lib/log"
+	"github.com/goharbor/harbor/src/pkg/artifact"
+)
+
+const (
+	// ArtifactTypeSBOM is the artifact type for SBOM
+	ArtifactTypeSBOM = "SBOM"
+	mediaType        = "application/vnd.goharbor.harbor.sbom.v1"
```

Comment:
according to the [pluggable-scanner-spec](https://github.com/goharbor/pluggable-scanner-spec/blob/master/data/spec/SBOM.md), the `artifactType` of Harbor SBOM Artifact is `application/vnd.goharbor.harbor.sbom.v1`, and the `mediaType` is `application/vnd.oci.image.manifest.v1+json`

Reply:
the artifact type is used to display the artifact type in the UI